README.md

```diff
@@ -323,7 +306,7 @@ Boolean, set to true to log LDAP auth.
 
 ####`manage_repos`
 
-Boolean, whether or not to manage package repositories.
+This option is deprecated in favor of repos_ensure, and is no longer honored.
```

Comment:
I think for deprecated parameters we can just remove the docs and comment in the code that it is deprecated.